app/helpers/youtube_helper.rb

```diff
@@ -1,8 +1,25 @@
 module YoutubeHelper
+  # Look below for expanded code
+  class << YoutubeHelper
+    include YoutubeHelper
+    include ActionView::Helpers::TagHelper
+  end
+
```

Comment:
Nice use of ruby object model! Opening up the included class and making the methods in the module class methods!